src/pybind/mgr/dashboard/frontend/src/testing/unit-test-helper.ts

```diff
@@ -101,3 +104,120 @@ export class PermissionHelper {
     });
   }
 }
+
+export class FormHelper {
+  form: CdFormGroup;
+
+  constructor(form: CdFormGroup) {
+    this.form = form;
+  }
+
+  /**
+   * Changes multiple values in multiple controls
+   *
+   * @param {object} values - {[controlName: string]:  changeToValue:any}
+   * @param {boolean} markAsDirty
+   */
```

Comment:
Can you please remove the redundant types in the doc strings? IDEs and documentation generators infer the types from the code. This will make the documentation and file a little bit more readable.